app/javascript/components/pages/PostNew.js

```diff
@@ -37,15 +37,20 @@ class ApartmentNew extends Component {
 	}
 
 	getNewPic = () => {
+
 		fetch(
-			'https://api.unsplash.com/photos/?client_id=-KQ11bQIym0-9gUNFtSQFdHMniZjFTIOp21lbrBFS1M'
+			'https://picsum.photos/v2/list?page=2&limit=1000'
```

Comment:
might need to limit to 90 if you're only generating a random index from 0-90 in the payload